src/tyr/actor.cc

```diff
@@ -62,10 +62,8 @@ actor_t::route(const std::string& request_str, const std::function<void()>* inte
   pimpl->loki_worker.route(request);
   // route between the locations in the graph to find the best path
   pimpl->thor_worker.route(request);
-  // get some directions back from them
-  pimpl->odin_worker.narrate(request);
-  // serialize them out to json string
-  auto bytes = tyr::serializeDirections(request);
+  // get some directions back from them and serialize
+  auto bytes = pimpl->odin_worker.narrate(request);
```

Comment:
i decided to collapse down the serialization into the `narrate` method in the odin worker. i did this mainly so  that i could count that time of doing serialization as part of the latency metric. if we'd rather track the odin maneuver gen and narration stuff separately from serialization i can undo this and move some of the bits around to accomplish it

Reply:
Hmm, it'd be interesting to know serialization cost separately, especially if we're exploring alternate response formats like pbf in the future. Haven't thought too much this change yet though nor have strong opinions on the change.

Reply:
i thought the same but didnt want to bother any more with crazy refactorings. now that i have it working i can take a second look at this